src/aap_eda/tasks/ruleset.py

```diff
@@ -55,24 +55,32 @@ def activate_rulesets(
 
 @job("default")
 def deactivate_rulesets(
+    user_deactivate: bool,
     activation_instance_id: int,
     deployment_type: str,
 ) -> None:
     instance = models.ActivationInstance.objects.get(pk=activation_instance_id)
     logger.info(f"Task started: Deactivate Activation ({instance.id})")
 
+    final_status = (
```

Comment:
Suggest to move this logic into `deactivate` function, it's confused and is not directly used here. Something like:

```
    ActivateRulesets().deactivate(
        instance,
        deployment_type,
        triggered_by_scheduler,
    )
```